apps/x509.c

```diff
@@ -217,7 +217,7 @@ int x509_main(int argc, char **argv)
                 goto opthelp;
             break;
         case OPT_KEYFORM:
-            if (!opt_format(opt_arg(), OPT_FMT_PEMDER, &keyformat))
+            if (!opt_format(opt_arg(), OPT_FMT_ANY, &keyformat))
```

Comment:
`OPT_FMT_PDE`? Here and for `cakeyform`.

Reply:
do you want a separate patch for `cakeyform` ?

Reply:
pushed V2 which is also rebased to master. But master fails in appveyor. so my V2 also fails

Reply:
rebased to a fixed master. now it passes.